doc/classes/Input.xml

```diff
@@ -399,8 +399,8 @@
 			<param index="0" name="duration_ms" type="int" default="500" />
 			<param index="1" name="amplitude" type="float" default="-1.0" />
 			<description>
-				[b]Note:[/b] While [code skip-lint]amplitude[/code] expects a value between 0 and 1, -1 does the default amplitude for the device.
 				Vibrate the handheld device for the specified duration in milliseconds.
+				You can control the strength of the vibration by using [param amplitude]. It must be a value between [code]0.0[/code] and [code]1.0[/code], or [code]-1.0[/code], which means the default vibration strength of the device.
```

Comment:
Alternative proposition.
```suggestion
				[param amplitude] is the strength of the vibration, as a value between [code]0.0[/code] and [code]1.0[/code]. If set to [code]-1.0[/code], the default vibration strength of the device is used.
```

Reply:
Either option is fine with me. I tend to agree that it's good to avoid "You" if we can. WDYT @timothyqiu?